doc/source/reference/commands/select.rst

```diff
@@ -1088,6 +1089,84 @@ more liked entries.
+   request to enable Apache Arrow to the package provider.
+
+:ref:`select-drilldown` , :ref:`drilldowns <select-advanced-drilldown-related-parameters>` and :ref:`slices <select-slice-related-parameters>` 
+are executed in parallel when this parameter is specified ``-1`` or ``2`` or more.
+
+In a default setting, ``drilldown``, ``drilldowns`` and ``slices`` are executed in serial.
+In other words, a next process is executed after a current process is finished.
+So, queries tend to take a long time if there are a lot of ``drilldown``, ``drilldowns`` and ``slices``.
+
+``n_workers`` enables to execute independent ``drilldown``, ``drilldowns`` and ``slices`` in parallel.
+The execution time of the total sum of processes can be shourtend by executing them in parallel.
+This parallel execution is done for each ``select`` command.
+
+"independent" means not using ``dorilldowns.table`` to reference the results of other drilldowns or slices.
+
+If there are dependencies as same meaning as using ``dorilldowns.table``, it wait for finish the dependent drilldowns or slices.
+Therefore, the degree of parallelism is reduced if they have dependencies.
+
+Executing in parallel means using multiple CPUs at the same time.
+If executing in parallel without free CPU resource, it may actually slow down the execution time.
+This is because they have to wait for the other process being executed by the target CPU to finish.
+
+It depends on a system configuration whether or not there are free CPU resources and how many ``n_workers`` should be specified.
+
+For example, consider using :doc:`/reference/executables/groonga-server-http` on a system with 6 CPUs.
+
+:doc:`/reference/executables/groonga-server-http` allocates 1 thread (= 1CPU) for each request.
+
+When the average number of concurrent connections is 6, there are no free CPU resources because 6 CPUs are already in use.
+All the CPU is used to process each request.
+
+When the average number of concurrent connections is 2, there are 4 free CPU resources because only 2 CPUs are already in use.
+When specifying ``2`` for ``n_workers``, the ``select`` command will use at most 3 CPUs, including the thread for processing requests.
+Therefore, if two ``select`` commands with ``2`` specified for ``n_workers`` are requested at the same time,
+they will use at most 6 CPUs in total and will be processed fastly by using all of the resources.
+When specifying greater than ``2``, the degree of parallelism can be higher than the CPU resources, so it may actually slow down the execution time.
+
+``n_workers`` behaves as follows depending on the specified value.
+
+* When specifying ``0`` or ``1``
+
+  * Executes the `select` command in serial
+
+* When specifying ``2`` or more
+  
+  * Executes the `select` command in parallel with at most the specified number of threads.
+
+* When specifying ``-1`` or less
+  
+  * Executes the `select` command in parallel with the threads of at most the number of CPU cores.
```

Comment:
It seems that `.. list-table::` is better than nested itemizations because the 2-level itemizations have only one item.

I'll merge this as-is but we can improve this as a follow-up task.